mllib/src/main/scala/org/apache/spark/ml/feature/RFormula.scala

```diff
@@ -248,6 +248,18 @@ class RFormulaModel private[feature](
       !columnNames.contains($(labelCol)) || schema($(labelCol)).dataType == DoubleType,
       "Label column already exists and is not of type DoubleType.")
   }
+
+  /**
+   * Get the original array of labels if exists.
+   */
+  private[ml] def getOriginalLabels: Option[Array[String]] = {
```

Comment:
Should we add a ```IndexToString``` transformer at the end of the ```PipelineModel```? I think it would be more general. Other functions such as ```glm``` with "binomial" family should also do the same work.